applications/DEMApplication/DEM_application.cpp

```diff
@@ -313,6 +313,7 @@ KRATOS_CREATE_VARIABLE(double, PARTICLE_ROTATIONAL_KINEMATIC_ENERGY)
 KRATOS_CREATE_VARIABLE(double, PARTICLE_GRAVITATIONAL_ENERGY)
 KRATOS_CREATE_VARIABLE(double, PARTICLE_INELASTIC_VISCODAMPING_ENERGY)
 KRATOS_CREATE_VARIABLE(double, PARTICLE_INELASTIC_FRICTIONAL_ENERGY)
+KRATOS_CREATE_VARIABLE(double, PARTICLE_INELASTIC_ROLLRESIST_ENERGY)
```

Comment:
the name is already long, you can write the full name I think

Reply:
Changed to full name